sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/scan/AbstractScan.java

```diff
@@ -282,6 +282,9 @@ protected void setupBasicConfiguration() throws ClientApiException {
         // enable all passive scanner rules by default
         // null specifies the default scan policy
         clientApi.ascan.enableAllScanners(null);
+
+        // use firefox in headless mode by default
+        clientApi.ajaxSpider.setOptionBrowserId("firefox-headless");
```

Comment:
Is there a command line argument for deciding what browser should be used? It would be nice to be able to switch between browsers. In any case, make a new issue for this feature, if you agree, that it should be possible to change the browser used for scanning.